src/main/scala/scalaz/parsers/transform.scala

```diff
@@ -5,7 +5,18 @@ import scalaz.parsers.implicits.monadKleisliCategory
 import scalaz.tc.ProfunctorClass.DeriveDimap
 import scalaz.tc._
 
-trait TransformClass[=>:[_, _]] extends StrongClass[=>:] with CategoryClass[=>:] {
+trait CategoryClassOps[=>:[_, _]] extends { self: CategoryClass[=>:] =>
+
+  implicit class ArrowOps[B, C](f: B =>: C) {
+    def ∘ [A](g: A =>: B): A =>: C = compose[A, B, C](f, g)
+  }
+
+}
+
+trait TransformClass[=>:[_, _]]
+    extends StrongClass[=>:]
+    with CategoryClass[=>:]
+    with CategoryClassOps[=>:] {
```

Comment:
I think extending `CategoryClassOps` defeats the purpose of having extension methods defined outside of the `trait TransformClass`, i.e. now `def ∘` is just an unusually defined method of `trait TransformClass`.